.github/workflows/documentation.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
       - name: install dependencies
         run: |
-          pip install -r requirements/development.txt
+          pip install . -r requirements/development.txt
```

Comment:
Why the `.` out of interest?

Reply:
conf.py now gets the version info from breathe as an installed package. That's what the door does. This should affect RTD as the readthedocs.yaml already tells RTD to install breathe as well.